app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use App\Models\Role;
+use App\Http\Requests\UserAddRequest;
+use App\Http\Requests\UserEditRequest;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Hash;
+
+class UserController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $users = User::all();
+        $searchKeyWord = "";
+
+        return view($this->path_to_view . 'listUser', compact(['users', 'searchKeyWord']));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view($this->path_to_view . 'addUser');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(UserAddRequest $request)
+    {
+        $password = Hash::make($request->password);
+        $user = new User;
+        $user = array('name' => $request->name, 'email' => $request->email,
+        'password' => $password, 'role_id' => $request->role_id);
+        $user = User::create($user);
```

Comment:
```suggestion
        $user = new User;
        $user = array('name' => $request->name, 'email' => $request->email,
        'password' => $password, 'role_id' => $request->role_id);
        $user = User::create($user);
```
Convention sai nè, ko dùng hàm array, dùng [].
Thứ 2: sao ko dùng create với array dữ liệu như trên đi, ghi đè 3 lần làm gì?